frontend/src/components/organisms/EventCard.tsx

```diff
@@ -3,6 +3,12 @@ import IconText from "@/components/atoms/IconText";
 import Button from "@/components/atoms/Button";
 import IconButton from "@mui/material/IconButton";
 import MoreVertIcon from "@mui/icons-material/MoreVert";
+import { Card, Icon } from "@mui/material";
+import LocationOnIcon from "@mui/icons-material/LocationOn";
+import ViewEvents from "./ViewEvents";
+import EventRegisterForm from "./EventRegisterForm";
+import { FaCarrot } from "react-icons/fa";
+import WatchLaterIcon from "@mui/icons-material/WatchLater";
```

Comment:
I wasn't able to find the specific carrot icon (for the title) using MUI, so I had to look for it in react-icons. Would it be okay if I changed the icon to something similar instead so that I could import fully from MUI icons?